JSDemos/Demos/Charts/PiesWithEqualSize/description.md

```diff
@@ -1 +1,3 @@
-When several PieChart components are displayed side by&nbsp;side, their pies may differ in&nbsp;size. To&nbsp;eliminate these differences, you can collect all **PieCharts** in&nbsp;a&nbsp;single size group by&nbsp;setting their **sizeGroup** property to&nbsp;identical values.
\ No newline at end of file
+To display multiple PieChart components that share [series](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/series/) and [legend](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/legend/) settings, you can declare an object with options and use this object in every PieChart.
+
+When you display PieChart components side by side, their pies may differ in size. make their sizes identical, you can join all charts in a size group. To do it, set the [sizeGroup](/Documentation/ApiReference/UI_Components/dxPieChart/Configuration/#sizeGroup) property of every chart to the same value.
```

Comment:
Several errors here. Use my notes from above to rewrite this section and others like it.